f/radio/fn_radioCheckChannels.sqf

```diff
@@ -5,9 +5,6 @@
 This function checks what channels a player should have access to. It's called by event handlers set up by fn_radioAddHandlers.sqf.
 =========================== */
 
-// Clients wait for the server to finish with setup
-waitUntil {!(isNil "f_var_radioChannelsUnified")};
```

Comment:
Why was this removed? I think we do need to wait for the server to initialize and broadcast all the `f_var_radio` variables here

Reply:
The server broadcasts the publicVariable before remoteExec'ing the next function to the client. According to Dedmen, those messages are guaranteed to be handled in order of transmission, so the client will always have the variables by the time it runs the function.

Reply:
I'm not sure I understand.
Here, the client calls this function directly. There is no remoteExec involved:
https://github.com/folkarps/F3/blob/72f33d443dec5fc1b9ae18359192265c5caaf311/f/radio/f_radioChannels.sqf#L106

Reply:
Confused it with something else. I've replaced that waitUntil as required, but it is now located in f_fnc_radioAddHandlers. It can't be located in f_fnc_radioCheckChannels because that function can be unscheduled, where waitUntil doesn't work properly.